src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -320,6 +321,8 @@ protected Object readResolve() throws ObjectStreamException {
         // ensure escapeHatchSecret is encrypted
         this.setEscapeHatchSecret(this.escapeHatchSecret);
         try {
+            // validate this option in FIPS env or not
+            this.setEscapeHatchEnabled(this.escapeHatchEnabled);
```

Comment:
After changing the setter to return a `FormException` we should change how compliance is checked here, and launch a `IllegalArgumentException` if not compliant

Reply:
actually this is part of the code using this https://github.com/olamy/oic-auth-plugin/blob/74ad0ec67a4cda6a8bc075cb4a2631e024845418/src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java#L346

Reply:
actually this is part of the code (in the try catch) using this https://github.com/olamy/oic-auth-plugin/blob/74ad0ec67a4cda6a8bc075cb4a2631e024845418/src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java#L346

Reply:
done with df6e4dd